projects/libreoffice/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -eu
+
+#shuffle CXXFLAGS -stdlib=libc++ arg into CXX as well because we use
+#the CXX as the linker and need to pass -stdlib=libc++ to build
+export CXX="$CXX -stdlib=libc++"
+#similarly force the -fsanitize etc args in as well as pthread to get
+#things to link successfully during the build
+export LDFLAGS="$CFLAGS -lpthread"
+
+cd $OUT
+#build under $OUT cause its the only place with enough space to do so
```

Comment:
hm. not sure if I understand where does this size difference come from. In both case it ends up on the same filesystem.

As for using $OUT rather than $WORK to cache build results, did you use helper.py to run it? I guess we can mount $WORK in the script to a local folder so that it is preserved automatically too.

All in all I'd prefer if you did a build inside a $WORK. You can always bind /work to local dir while running a build if you want it to be cached.